### PR TITLE
Some first feedback fixes

### PR DESCRIPTION
**Pull request by lmcgrath**
This fixes the following checked issues:

**Primary asks:**
❌ spreading players at world spawn--we should really either do a creative build at spawn or write a mod to randomly spawn the player if they're either new or don't have a bed set up
✅ too many coppers and other ores fixed, we're going to be reliant primarily on immersive engineering mod oregen and I tweaked it to spawn more frequently because it's pretty light by default
❌ removing mekanism mod--WONT FIX, we don't get a lot of ores in this pack so I'm keeping the OP duplication, anything beyond 3x duplication is a massive time and resource investment so I'm considering this "balanced", there's also generators (like fusion reactors!) and other tools added that I wanna play with
❌ adding randompatches mod (already in master)
✅ oneprobe readme/config no longer given to newly spawned players
✅ change caster focus keybinding to "Y" (done by adding the defaultoptions mod)
✅ adding malisisdoors mod
✅ removing graves (graves came from the erebus mod and have been configured not to be used, there's also no death compass given now either)
✅ weird ftb utils mod button layout fixed by removing the mod
✅ /back /spawn and related commands came from ftb utils mod so they're now gone
✅ removing just-enough-buttons mod to get rid of all those other weird buttons around the weird ftb utils mod buttons because they all require op-level permissions in order to be usable
✅ adding akashic tome mod, must be crafted in order to be used (book + bookshelf)

**Additional stuff:**
✅ lengthening seasons from 21 days to 45 days
✅ adding defaultoptions mod, it allows for defaults for server config, controls, and video/audio settings to be set on the pack so we can distribute the modpack in a tuned configuration that we know works out of the box
✅ updating ore distribution list for just-enough-resources mod (this needs to be done anytime resources are changed, through configuration or by adding/removing mods)
✅ adding reauth mod so we don't have to restart the game if our session expires
✅ adding stackie mod so xp orbs clump together
✅ adding fence jumper mod
✅ adding login shield mod because logging in and dying before you're actively playing really sucks
✅ adding portal gun mod because I know vlad ❤️it, I also turned down the settings to use the ugly portals you can't see through because the fancy ones drop FPS below 10
✅ adding crafting tweaks mod for the handy "put this back in my inventory" button
✅ upgraded wizardry to 0.10.6 (was 0.10.0)
✅ removed just-enough-resources mod because scanning dimensions for resources takes FOREVER and I don't want to have to do that each time we modify resources (advanced rocketry mod adds literally hundreds of dimensions and the only way to exclude them is to blacklist, which is really clunky)
✅ advanced rocketry: allowing 0g space stations, letting players to spawn on planets with a bed, and orbital laser to mine from planets they're orbiting
✅ nuclearcraft: turning on radiation
✅ extrautils2: removing chunk loading ward block
✅ theoneprobe: turning off mob health and potion effects
✅ adding rats mod
✅ adding nutrition mod

### CODE REVIEW

File: overrides/config/nuclearcraft.cfg [hunk 1]
```diff
@@ -0,0 +1,1590 @@
...
+    B:passive_permeation=true
+
+    # Will machines produce particle effects while running?
+    B:processor_particles=true
+}
+
+
+radiation {
+    # Will nuclear radiation exist in the world?
+    B:radiation_enabled=false
```
Comment (LordLandon) [inline overrides/config/nuclearcraft.cfg]: this one is also fun to flip maybe?

File: overrides/config/theoneprobe.cfg [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    I:showItemDetailThresshold=4
+
+    # Show lever/comparator/repeater settings (0 = not, 1 = always, 2 = sneak) [range: 0 ~ 2, default: 1]
+    I:showLeverSetting=1
+
+    # Show time to adulthood for baby mobs (0 = not, 1 = always, 2 = sneak) [range: 0 ~ 2, default: 1]
+    I:showMobGrowth=1
+
+    # Show mob health (0 = not, 1 = always, 2 = sneak) [range: 0 ~ 2, default: 1]
+    I:showMobHealth=1
```
Comment (LordLandon) [inline overrides/config/theoneprobe.cfg]: how do you feel about turning this to 0?

File: overrides/config/theoneprobe.cfg [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+    # Show mob potion effects (0 = not, 1 = always, 2 = sneak) [range: 0 ~ 2, default: 2]
+    I:showMobPotionEffects=2
```
Comment (LordLandon) [inline overrides/config/theoneprobe.cfg]: and this?

**Comment by LordLandon** [comment]
also let's add https://minecraft.curseforge.com/projects/mouse-tweaks so you can shift-drag over slots to move em, mouse wheel to move items granularity

File: overrides/config/advRocketry/advancedRocketry.cfg [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+
+    # The largest size a space station can be.  Should also be a power of 2 (512, 1024, 2048, 4096, ...).  CAUTION: CHANGING THIS OPTION WILL DAMAGE EXISTING STATIONS!!!
+    I:SpaceStationBuildRadius=1024
+
+    # how many millibuckets/t are required to keep the terraformer running
+    I:TerraformerFluidConsumeRate=40
+    B:TerraformerRequiresFluids=true
+
+    # If true players will respawn near beds on planets IF the spawn location is in a breathable atmosphere
+    B:allowPlanetRespawn=false
```
[the remark this answers is not in the document]
Comment (lmcgrath) [reply overrides/config/advRocketry/advancedRocketry.cfg]: :+1:

File: overrides/config/advRocketry/advancedRocketry.cfg [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+        oreTin
+        oreRedstone
+        oreDiamond
+     >
+
+    # True if the ores in laserDrillOres should be a blacklist, false for whitelist
+    B:laserDrillOres_blacklist=false
+
+    # If true the orbital laser will actually mine blocks on the planet below
+    B:laserDrillPlanet=false
```
[the remark this answers is not in the document]
Comment (lmcgrath) [reply overrides/config/advRocketry/advancedRocketry.cfg]: :+1:

File: overrides/config/nuclearcraft.cfg (hunk 1 above)
Comment (lmcgrath) [reply overrides/config/nuclearcraft.cfg]: :+1:

File: overrides/config/advRocketry/advancedRocketry.cfg [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+    B:allowPlanetRespawn=false
+
+    # EXPERIMENTAL: If set to true allows contruction and usage of the terraformer.  This is known to cause strange world generation after successful terraform
+    B:allowTerraforming=false
+
+    # If true dimensions not added by AR can be terraformed, including the overworld
+    B:allowTerraformingNonARWorlds=false
+
+    # If true players will be able to completely disable gravity on spacestation.  It's possible to get stuck and require a teleport, you have been warned!
+    B:allowZeroGSpacestations=false
```
[the remark this answers is not in the document]
Comment (lmcgrath) [reply overrides/config/advRocketry/advancedRocketry.cfg]: :+1:

File: overrides/config/extrautils2.cfg [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+enabled {
+    B:AnalogCrafter=true
+    B:AngelBlock=true
+    B:AngelRing=true
+    B:BagOfHolding=true
+    B:BiomeMarker=true
+    B:Book=true
+    B:Boomerang=true
+    B:ChickenRing=true
+    B:ChunkLoader=true
```
[the remark this answers is not in the document]
Comment (lmcgrath) [reply overrides/config/extrautils2.cfg]: :+1:

File: overrides/config/theoneprobe.cfg (hunk 2 above)
Comment (lmcgrath) [reply overrides/config/theoneprobe.cfg]: :+1:

File: overrides/config/theoneprobe.cfg (hunk 3 above)
Comment (lmcgrath) [reply overrides/config/theoneprobe.cfg]: :+1:

**Comment by lmcgrath** [comment]
I tweaked those settings and added mouse tweaks :)

**Comment by LordLandon** [comment]
LGTM

**Pull request merged**